editors/code/src/rust-analyzer-api.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * This file mirrors `crates/rust-analyzer/src/req.rs` declarations.
+ */
+
+import { RequestType, TextDocumentIdentifier, Position, Range, TextDocumentPositionParams, Location, NotificationType, WorkspaceEdit } from "vscode-languageclient";
```

Comment:
import * as lc?

Reply:
Followed the same import pattern we have in `req.rs`

Reply:
Okay, done